config/config.yaml.dist

```diff
@@ -362,6 +362,11 @@ dex:
 #
 #                # See https://github.com/kubernetes-sigs/kubefed/tree/master/charts/kubefed for details
 #                values: {}
+#
+#    posthook:
+#        ingressconrtroller:
```

Comment:
```suggestion
#        ingresscontroller:
```

Reply:
The configuration should probably follow configuration structure for other features: chart name and version should be configurable.

Reply:
Or just:

```suggestion
#        ingress:
```